tests/integration/__init__.py

```diff
@@ -182,8 +182,19 @@ def set_default_cass_ip():
         Cluster.__init__.__func__.__defaults__ = tuple(defaults)
 
 
-def get_default_protocol():
+def set_default_beta_flag_true():
```

Comment:
This set the flag `allow_beta_protocol_version` to `True` by default so we don't have to pass the argument. This is temporary since it looks like V5 won't be beta in Cassandra 4.0 although it is now.